src/components/style/interactive/button.css

```diff
@@ -141,6 +153,7 @@
 .inverted .fdsButton--ghost,
 .inverted .fdsButton--outlined {
   color: var(--color-white);
+  background: transparent;
```

Comment:
So on light colors, the ghost and outline buttons will have white backgrounds, and on dark backgrounds, they will be transparent?

Maybe more of a design question, but are we sure we want a `ghost` button on a `sky` background to be white?

Reply:
@akdetrick I think I've resolved all the issues here, and Asaf has addressed them in a way that I think makes sense. Essentially, all primary color themes no longer have transparency. You can now consider the outlined button to be a `white` button (similar to blue). It's not necessary used on inverted contexts. The only other true tranparent buttons will be the monochrome, and ghost ones. I've made the necessary modifications, but happy to show my storybook and talk through it with you.

Reply:
For clarity, would it make sense to put this `background: transparent` up at line 106 where `ghost` is initially declared, or does it need to be here for the cascade